firebase-database/src/test/java/com/google/firebase/database/MapperTest.java

```diff
@@ -1380,54 +1380,28 @@ public void serializingListsWorks() {
   @Test
   public void serializingMapsWorks() {
     MapBean bean = new MapBean();
-    bean.values =
```

Comment:
Parameter list initialization is disallowed because it leads to memory issues. We can use the ImmutableFoo.of methods in unit tests or the ImmutableFoo builders in code with a lower target JRE version.